stactools_pipelines/pipelines/amazonia_1/test_app.py

```diff
@@ -46,7 +46,7 @@ def test_handler(
     handler(event=sqs_event, context={})
     get_token.assert_called_once()
     create_item.assert_called_once_with(
-        asset_href="s3://amazonia-pds/AMAZONIA1/WFI/035/020/"
+        asset_href="s3://brazil-eosats/AMAZONIA1/WFI/035/020/"
```

Comment:
We might need to update existing STAC records because the `amazonia-pds` bucket does not seem to exist anymore!
```shell
$ aws s3 ls --no-sign-request s3://amazonia-pds/

An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
```

Reply:
There's a ticket for this that I've been finishing up!